lib/timeline/locales.js

```diff
@@ -11,7 +11,8 @@ exports['en_US'] = exports['en'];
 require('moment/locale/it');
 exports['it'] = {
   current: 'attuale',
-  time: 'tempo'
+  time: 'tempo',
+  deleteSelected: 'Cancella la selezione',
```

Comment:
`elimina l'elemento selezionato`

Reply:
This is taken from https://github.com/visjs/vis-network/blob/505ef4e9aa5c500a7597523c0eed7f0ad27d45c0/lib/network/locales.js#L59
If this is wrong, vis-network is also wrong